.gitignore

```diff
@@ -9,3 +9,7 @@ tmp/
 /.tox
 /docs/build
 .idea/
+.eggs/
+.hypothesis/
```

Comment:
Please remove trailing slashes, as they have no effect in .gitignore.  (Also for 'env*/')

On the other hand, consider adding a _leading_slash, to each entry where you expect that name to be at top level in the repo.